sdk/servicebus/azure-servicebus/tests/test_queues.py

```diff
@@ -208,14 +223,17 @@ def test_queue_by_queue_client_conn_str_receive_handler_peeklock(self, servicebu
             assert count == 10
 
     # @pytest.mark.skip(reason="TODO: Pyamqp Message Serialization Error")
+    # TODO: pyamqp failing w/ assert 1 == 6
```

Comment:
Do we plan to investigate in this PR?

Reply:
yes this has been fixed, just forgot to remove this TODO :P